### PR TITLE
Pi: Switching to text template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

Ref: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

###### People to notify (@mention interested parties)

@NaveenKarippai 

---

Instant Answer Page: https://duck.co/ia/view/pi

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @jmvbxx

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Pi.pm [hunk 1]
```diff
@@ -23,11 +23,14 @@ handle query_raw => sub {
...
     my $answer = substr $PI, 0, ( $1 + 2 );
-    return $answer,
-    structured_answer => {
-        input     => [],
-        operation => ["First ".$+{decimal}." digits of Pi"],
-        result    => $answer
+    return $answer, structured_answer => {
+        data => {
+            title => $answer,
+            subtitle => "First ".$+{decimal}." digits of Pi"
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Pi.pm]: `First $+{decimal} digits of Pi`

File: t/Pi.t [hunk 2]
```diff
@@ -10,45 +10,28 @@ use utf8;
...
-		input => [],
-		operation => ["First 6 digits of Pi"],
-		result => "3.141592"
-	}),
+	'pi 23' => build_test("3.14159265358979323846264",23),
+	'π 8' => build_test("3.14159265", 8),
+	'12 digits of pi' => build_test("3.141592653589", 12),
+    '12 digits of π' => build_test("3.141592653589", 12),
+    'pi to 6 digits' => build_test("3.141592", 6),
+    'π to 6 digits' => build_test("3.141592", 6),
```
Comment (GuiltyDolphin) [inline t/Pi.t]: Something has gone awry with the formatting here!

Comment (mintsoft) [inline t/Pi.t]: That would be the wrong editing profile loaded :)

**Comment by GuiltyDolphin** [comment]
@mintsoft Surely whichever editor you use can *automatically* update preferences based on filetype+context?

Thanks :wink:

**Comment by mintsoft** [comment]
@GuiltyDolphin I write perl everywhere! This is the only place that insists on evil spaces :imp:

**Comment by GuiltyDolphin** [comment]
@mintsoft Hehe, 4 spaces on ZCI, 2 spaces in Haskell, 2 in Idris, tabs in other DuckDuckGo... Thank God for Vim :wink:

**Pull request merged**